src/Raven.Server/Documents/Replication/IncomingReplicationHandler.cs

```diff
@@ -1380,8 +1380,8 @@ protected override long ExecuteCmd(DocumentsOperationContext context)
                                  _replicationInfo.ReplicatedAttachmentStreams.Count == handledAttachmentStreams.Count,
                         "We should handle all attachment streams during WriteAttachment.");
 
-                    Debug.Assert(string.IsNullOrEmpty(context.LastDatabaseChangeVector) == false, $"database: {context.DocumentDatabase.Name}; context.LastDatabaseChangeVector = {context.LastDatabaseChangeVector ?? "null"};");
-
+                    Debug.Assert(context.LastDatabaseChangeVector != null);
```

Comment:
How come that this happens?

Reply:
I put an explanation in YT: https://issues.hibernatingrhinos.com/issue/RavenDB-17066

Reply:
Please add a comment here explaining why this is okay to be ""

Reply:
Done